test/external-modules/error-query/src/javaRestTest/java/org/elasticsearch/test/esql/EsqlPartialResultsIT.java

```diff
@@ -35,8 +34,9 @@
 public class EsqlPartialResultsIT extends ESRestTestCase {
     @ClassRule
     public static ElasticsearchCluster cluster = ElasticsearchCluster.local()
-        .distribution(DistributionType.DEFAULT)
         .module("test-error-query")
+        .module("x-pack-esql")
+        .module("x-pack-ilm")
```

Comment:
esql requires ml which implicitly requires ilm. Perhaps that dependency should be explicit? 🤷